lib/rom/sql/relation/reading.rb

```diff
@@ -1036,7 +1038,7 @@ def lock_clause(mode: :update, skip_locked: false, of: nil, wait: nil)
           stmt << ' OF ' << Array(of).join(', ') if of
 
           if skip_locked
-            raise ArgumentError, "SKIP LOCKED cannot be used with (NO)WAIT clause" if !wait.nil?
+            raise ArgumentError, 'SKIP LOCKED cannot be used with (NO)WAIT clause' if !wait.nil?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Favor `unless` over `if` for negative conditions.](https://app.codacy.com/app/rom-rb/rom-sql/pullRequest?prid=4731477)